server/queries/admin/allPharmacies.ts

```diff
@@ -0,0 +1,21 @@
+import { Pharmacy } from '../../models';
+
+const getAllPharmaciesGeneral = async (status: any, page: any) =>{
```

Comment:
edit type for ( status, page )

Reply:
We are getting status and page form query and it has many types and you can not restrict it to only having one type.
So both do the job whether being type any or assigning all the types that query you are getting from the request.